gradle.properties

```diff
@@ -16,4 +16,7 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
-android.enableJetifier=true
\ No newline at end of file
+android.enableJetifier=true
+android.defaults.buildfeatures.buildconfig=true
+android.nonTransitiveRClass=false
+android.nonFinalResIds=false
```

Comment:
what are those properties?

Reply:
They are also automatically generated files.
android.nonFinalResIds=false
Its used to keep ResIds as a **final** variable and not to change during compile time

android.nonTransitiveRClass=false
generates R classes with non-final fields by default.

Reply:
ok, if you think it's useful I'm good :)